Detections/AzureDiagnostics/TimeSeriesKeyvaultAccessAnomaly.yaml

```diff
@@ -55,7 +55,8 @@ query: |
   | where requestUri_s != "None" and isnotempty(requestUri_s)
   | where ResourceType == "VAULTS" and ResultType == "Success"
   | where OperationName in (OperationList)  
-  | summarize PerOperationCount=count() by bin(TimeGenerated,1h), Resource, OperationName, id_s, CallerIPAddress, identity_claim_http_schemas_microsoft_com_identity_claims_objectidentifier_g, requestUri_s, clientInfo_s     
+  | summarize PerOperationCount=count(), TimeGeneratedMax=arg_max(TimeGenerated,*) by bin(TimeGenerated,1h), Resource, OperationName, id_s, CallerIPAddress, identity_claim_http_schemas_microsoft_com_identity_claims_objectidentifier_g, requestUri_s, clientInfo_s     
   ) on Resource, TimeGenerated 
-  | project AnomalyHour, Resource, CallerIPAddress, HourlyCount, baseline, anomalies, score, OperationName, id_s, identity_claim_http_schemas_microsoft_com_identity_claims_objectidentifier_g, requestUri_s, clientInfo_s, PerOperationCount
-  | extend timestamp = AnomalyHour, IPCustomEntity = CallerIPAddress, AccountCustomEntity = identity_claim_http_schemas_microsoft_com_identity_claims_objectidentifier_g
\ No newline at end of file
+  | project LatestAnomalyTime = TimeGeneratedMax , Resource, CallerIPAddress, HourlyCount, baseline, anomalies, score, OperationName, id_s, identity_claim_http_schemas_microsoft_com_identity_claims_objectidentifier_g, requestUri_s, clientInfo_s, PerOperationCount
```

Comment:
Any reason not to just go with the summarize here, versus the extra step of project?